src/core/gltf2importer/gltf2context_p.h

```diff
@@ -146,6 +146,9 @@ class Q_AUTOTEST_EXPORT GLTF2Context
     const QJsonDocument &json() const;
     void setJson(const QJsonDocument &doc);
 
+    const QVector<QString> &localFiles() const;
```

Comment:
why go for QStringList in overwritableFiles and QVector<QString> here?

Reply:
good question, hadn't thought about it. which one would be prefferred ? I think that I was influence by many things being QVector<foo> in GLTFContext  but on the other hand, QStringList has nice utilty methods...

Reply:
went for qstringlist in the latest commit